README.md

```diff
@@ -1,5 +1,11 @@
 # Paradox Conflict Detector
 
+## Fork
+
+Fixed issue with uncode encoding https://github.com/NicolasGrosjean/Paradox_Conflict_Detector/issues/1
+
+Fixed common files being detected as conflicts (readme.md, changelog.md)
+
```

Comment:
Could you remove this part to be merge on the main repository ?

Reply:
Done